tests/operator_test.go

```diff
@@ -1262,7 +1284,7 @@ spec:
 		// running a VM/VMI using that previous release
 		// Updating KubeVirt to the target tested code
 		// Ensuring VM/VMI is still operational after the update from previous release.
-		It("[release-blocker][sig-compute][test_id:3145]from previous release to target tested release", func() {
+		table.DescribeTable("[release-blocker][sig-compute][test_id:3145]from previous release to target tested release", func(updateOperator bool) {
```

Comment:
The cases here are quite important. I wonder if it is worth having them outside of a table to get rid of the if/else parts.

Also can we skip the full-update tests if no operator yaml is provided as input to not execute the test then?

Reply:
> I wonder if it is worth having them outside of a table to get rid of the if/else parts.

The if's aren't great, but the vast majority of the test code is shared. Duplicating this ~250 lines long test case seems worse...

 > Also can we skip the full-update tests if no operator yaml is provided as input to not execute the test then?
 
Right, since this anyway won't work without this flag value. Note that currently the flag has a non-empty default, so we'd need to pass an explicit empty flag to disable. Do you think we should default to empty (i.e., default to skip test)?

Reply:
> Right, since this anyway won't work without this flag value. Note that currently the flag has a non-empty default, so we'd need to pass an explicit empty flag to disable. Do you think we should default to empty (i.e., default to skip test)?

Yes empty as default sounds great.

> The if's aren't great, but the vast majority of the test code is shared. Duplicating this ~250 lines long test case seems worse...

Yes, they may still deserve their own room. But we can split them if for one of the two scenarios more cases come up and we want a separate table just for those.

Reply:
Very well. I'll work on splitting this.

Reply:
Changed in 8f404fc (this is the only actual change, the rest is just a rebase).